README.md

```diff
@@ -435,32 +435,56 @@ inductiva.tasks.list(5, status="success")
 
 ## Manage Resources
 
-**Inductiva API** provides a simple way to manage the hardware resources used to run the simulations. Users can launch virtual machines, list the available machines and terminate them. This is a feature available only to admins.
-In this way, users do not need to wait for their simulations in a queue and can have full control of the hardware used.
+**Inductiva API** provides a simple way to manage the hardware resources used to run simulations.
+This allows the user to have control over hardware that will be used exclusively for their experiments.
+Resources are controlled via machine groups. Users can create objects of type `MachineGroup`, and then use them to start and terminate the machines. Creating a `MachineGroup` requires specifying the type of Virtual Machine to use and the number of homogeneous machines that should be launched as part of the group. Currently, the available options for the `machine_type` are the ones available in the [Google Cloud Platform](https://cloud.google.com/compute/docs/machine-types). Once a `MachineGroup` is created, simply pass it as argument to your simulations, and they will be scheduled to run on those machines. Note that launching several machines in the group serves to run simulations in parallel, *i.e.*, one in each machine, and machines in the same machine group do not perform the same simulation cooperatively.
```

Comment:
"Resources are controlled via machine groups".
If you read this for the first time, this says nothing. In fact, if you remove this sentence it is ok. See if you can change this sentence for something more informative.

Reply:
argument to your simulations, and they --->  argument to your simulations, which will then be scheduled to run on those machines.

Reply:
Note that launching several machines in the group serves to run simulations in parallel, *i.e.*, one in each machine, and machines in the same machine group do not perform the same simulation cooperatively -->

Note that a MachineGroup is literally a group of individual machines that do not communicate with each other. In other words, a MachineGroup is not a computational cluster where the load of each simulation is divided over all machines of the cluster. ...